lib/src/test/java/com/thenewboston/utils/Mocks.kt

```diff
@@ -365,6 +367,19 @@ object Mocks {
         80,
         "",
     )
```

Comment:
Can we remove the redundant `cleanSuccess` and `cleanFailure` functions?

Reply:
These both functions for separate response for testing, I don't think we could remove it.